docs/contributing/process.md

```diff
@@ -0,0 +1,15 @@
+# Enso Team Process
+
+This document specify our core team workflow, described as a lifecycle of a task:
+
+* A newly created task should be appropriately described, but not estimated yet - it will be during
+ Backlog Refinement meeting. The task is put in "New Tasks" column. 
+* At the beginning of the sprint team leads put to the "To Refine" column all tasks which will be 
+ refined during the next Backlog refinement.
+* Each team member should read the descriptions of tasks in "To Refine" column and ask questions
+ and raise all concerns. All the conversation should be recorded in the issue's comments.  
+* During Backlog Refinement we confirm that the task description is clear and estimate it with
+ "Scrum poker" methodology. The task is then moved to the "Backlog" column. If it turns out that
```

Comment:
Just to clarify: my understanding of our current process is that we actually do a time estimate ("number of days needed to complete a task"), instead of "[Scrum Poker](https://en.wikipedia.org/wiki/Planning_poker)" which tends to estimate abstract story points. Is this meant as a change to the current way we estimate? If not we might want to clarify this here.